src/audio/pipeline.c

```diff
@@ -124,6 +124,7 @@ struct pipeline_walk_context {
 	void *comp_data;
 	const void (*buff_func)(struct comp_buffer *, void *);
 	void *buff_data;
+	bool skip_incomplete;
```

Comment:
Lets have a comment here explaining the rule and when and where it will be used.